rockcraft.yaml

```diff
@@ -65,3 +71,22 @@ parts:
       umount --recursive $CRAFT_OVERLAY/dev
       umount $CRAFT_OVERLAY/proc
       umount $CRAFT_OVERLAY/etc/resolv.conf
+  prometheus-exporter:
+    plugin: python
+    stage-packages:
+      - python3-venv
+    source: https://github.com/pyToshka/wazuh-prometheus-exporter.git
+    source-depth: 1
+    source-type: git
+    source-commit: 7ab31b3f73b0ce344bd7e4d0deb1d544ae1b7d1c
+    python-requirements: [requirements.txt]
+    override-build: |
+      cp $CRAFT_PART_SRC/*py $CRAFT_PART_INSTALL/
```

Comment:
I think this will only copy `main.py` and not `wazuh/*.py` from the original repository?

Reply:
Good catch. Fixed